### PR TITLE
Update reflect extension

### PR DESCRIPTION
**Pull request by theomjones**
## Description

- [x] Added support for multiple list options in comma separated format within preferences
- [x] Added parent list substring search to quick add as an optional argument

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/reflect/metadata/reflect-1.png
Comment (theomjones) [inline extensions/reflect/metadata/reflect-1.png:1]: Oops - sizing off, will fix

**Comment by pernielsentikaer** [comment]
Thanks for your contribution @theomjones 🔥 

Do you mind checking this @RyonHuedepohl

**Comment by pernielsentikaer** [comment]
@RyonHuedepohl are you still around?

**Comment by joshknowles** [comment]
@pernielsentikaer if Ryon is no longer active can someone else provide feedback approval, or does it need to be the original extension author?

**Comment by RyonHuedepohl** [comment]
My apologizes for not seeing this earlier, I will be taking a look now.

**Review by RyonHuedepohl — APPROVED** [review]
Looks great!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**